content/intro-to-writing-data.md

```diff
@@ -82,35 +28,35 @@ When you send a transaction to a Solana cluster, a Solana program is invoked wit
 
 As you might expect, `@solana/web3.js` provides helper functions for creating transactions and instructions. You can create a new transaction with the constructor, `new Transaction()`. Once created, then you can add instructions to the transaction with the `add()` method.
 
-Instructions can get complicated when working with custom programs. Fortunately, `@solana/web3.js` has convenience functions for some of Solana’s native programs and basic operations, like transferring SOL:
+One of those helper function is `SystemProgram.transfer()`, which makes an instruction for transferring SOL:
 
-```tsx
+```ts
 const transaction = new Transaction()
 
 const sendSolInstruction = SystemProgram.transfer({
-    fromPubkey: sender,
-    toPubkey: recipient,
-    lamports: LAMPORTS_PER_SOL * amount
+  fromPubkey: sender,
+  toPubkey: recipient,
+  lamports: LAMPORTS_PER_SOL * amount
 })
 
 transaction.add(sendSolInstruction)
 ```
 
-The `SystemProgram.transfer()` function requires that you pass as parameters:
+The `SystemProgram.transfer()` function requires:
 
 - a public key corresponding to the sender account
 - a public key corresponding to the recipient account
 - the amount of SOL to send in lamports.
 
-This function then returns the instruction for sending SOL from the sender to the recipient, after which the instruction can be added to the transaction.
+`SystemProgram.transfer()` returns the instruction for sending SOL from the sender to the recipient. The instruction can then be added to the transaction.
 
-Once created, a transaction needs to be sent to the cluster and confirmed:
+Once all the instructions have been added, a transaction needs to be sent to the cluster and confirmed:
 
-```tsx
+```ts
```

Comment:
I've had issues with `ts` working with markdown code blocks in the past. Just would like to confirm that this works. Alternatively I know `typescript` and `tsx` work (the tsx comes from drafting these in Notion and that's what Notion uses under the hood for their TS code blocks)